### PR TITLE
Create `@guardian/core-web-vitals`

### LINKED ISSUE #135: Remove `peerDependencies` from `@guardian/libs`
libs is a general purpose collection of useful things - if code is complicated enough that it needs peer deps, it should have it's own package

in this instance, it feels like a code smell that you need to install `web-vitals` to check whether something is a string

### PR DESCRIPTION
**Pull request by sndrs**
## What are you changing?

- removes core web vitals functionality from `@guardian/libs`
- moves it to a new `@guardian/core-web-vitals` package

## Why?

- fixes #135

### CODE REVIEW

**Review by mxdvl — APPROVED** [review]

**Comment by sndrs** [comment]
any idea what could be triggering this @mxdvl https://github.com/guardian/csnx/actions/runs/3299059309/jobs/5441972288#step:5:355?

File: libs/@guardian/core-web-vitals/src/index.test.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
+import * as libs from '@guardian/libs';
```
Comment (mxdvl) [inline libs/@guardian/core-web-vitals/src/index.test.ts:1]: I think this is the wrong import.

```suggestion
import { logger } from '@guardian/libs';
```

**Review by mxdvl — COMMENTED** [review]
Looking at this again, it seems the ReadMe is duplicated and slightly wrong.

File: libs/@guardian/core-web-vitals/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"rules": {}
+		},
+		{
+			"files": ["*.ts", "*.tsx"],
+			"rules": {}
+		},
+		{
+			"files": ["*.js", "*.jsx"],
+			"rules": {}
+		},
```
Comment (tjsilver) [inline libs/@guardian/core-web-vitals/.eslintrc.json:19]: What do these overrides do?

File: libs/@guardian/core-web-vitals/README.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
+# `@guardian/core-web-vitals`
+
+[![npm (scoped)](https://img.shields.io/npm/v/@guardian/core-web-vitals)](https://www.npmjs.com/package/@guardian/core-web-vitals)
+[![ES version](https://badgen.net/badge/ES/2020/cyan)](https://tc39.es/ecma262/2020/)
+[![npm type definitions](https://img.shields.io/npm/types/@guardian/core-web-vitals)](https://www.typescriptlang.org/)
+[![Coverage Status](https://coveralls.io/repos/github/guardian/libs/badge.svg)](https://coveralls.io/github/guardian/libs)
```
Comment (mxdvl) [inline libs/@guardian/core-web-vitals/README.md:6]: This coverage is out of date and not applicable to this repo, should we remove it entirely?

**Review by tjsilver — DISMISSED** [review]
Ran `make validate` locally and the project was built successfully 👍

File: libs/@guardian/core-web-vitals/.eslintrc.json (hunk 2 above)
Comment (sndrs) [reply libs/@guardian/core-web-vitals/.eslintrc.json:19]: nothing here, they're duplicates of the other configs though - they mainly provide ways in for people in the future. theyc oudl go though

File: libs/@guardian/core-web-vitals/README.md (hunk 3 above)
Comment (sndrs) [reply libs/@guardian/core-web-vitals/README.md:6]: good point - will remove from libs too

**Pull request merged**